web/quiz_views.py

```
@@ -360,87 +584,149 @@
+                    student_score = correct_points - wrong_points
+
+                    if student_score < 0:
+                        student_score = 0
+                    elif student_score >= question_obj.points:
+                        student_score = question_obj.points
+
+                    res = isinstance(student_score, float)
+                    if res:
+                        student_score = round(student_score, 1)
+
+                    User_answer_true_or_false = True if student_score >= (question_obj.points / 2) else False
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)